lib/open_project/backlogs/hooks.rb

```diff
@@ -80,40 +80,43 @@ def view_work_packages_form_details_bottom(context = {})
 
       snippet << %(<div id="backlogs-attributes" class="attributes">)
 
-      if work_package.is_story?
-        snippet << '<div class="attribute_wrapper">'
-        snippet << context[:form].text_field(:story_points, :size => 3, :class => 'short')
-        snippet << '</div>'
-
-        if work_package.descendants.length != 0
-
-          snippet << javascript_tag(<<-JS)
-            (function($) {
-              $(document).ready(function() {
-                $('#issue_estimated_hours').attr('disabled', 'disabled');
-                $('#issue_remaining_hours').attr('disabled', 'disabled');
-                $('#issue_done_ratio').attr('disabled', 'disabled');
-                $('#issue_start_date').parent().hide();
-                $('#issue_due_date').parent().hide();
-              });
-            }(jQuery))
-          JS
-        end
-      end
-
-      snippet << '<div class="attribute_wrapper">'
-      snippet << context[:form].text_field(:remaining_hours, :size => 3, :class => 'short', :placeholder => l(:label_hours))
-      snippet << '</div>'
-
       params = context[:controller].params
       if work_package.is_story? && params[:copy_from]
```

Comment:
If I see this correctly the outer div (line 81: `<div id="backlogs-attributes" class="attributes">`) is only neccessary if there is the `copy_from` parameter present, right? Because the "normal" case is completely covered using the `WorkPackageHelper`.

If that is true I'd prefer to move that part of the markup into the conditional statement, to keep the vanilla form clear of additional markup.